common/lib/xmodule/xmodule/templates/html/downloads.yaml

```diff
@@ -0,0 +1,10 @@
+---
+metadata:
+  display_name: Downloads
+  data: |
+    <div class="div-info"><img class="icon-image" src="/static/!downloads.png" alt="" />
```

Comment:
Nit: I'm guessing the "!" in the `src` field got in there by mistake but it's preventing the image from loading currectly. So please correct it.

Reply:
Thanks! Fixed.